notebooks/02_model/lightfm_deep_dive.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>would it be possible to have all the imports on the first cell and then the parameters on a different cell? We are doing that in the rest of the notebook. Also we use the parameters cell to execute the notebook tests, please see more details here: <a href="https://github.com/microsoft/recommenders/blob/master/tests/README.md#how-to-create-tests-on-notebooks-with-papermill" target="_blank">https://github.com/microsoft/recommenders/blob/master/tests/README.md#how-to-create-tests-on-notebooks-with-papermill</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done, also added parameters cell.